src/EtherFiAdmin.sol

```diff
@@ -53,17 +49,24 @@ contract EtherFiAdmin is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     uint16 public postReportWaitTimeInSlots;
     uint32 public lastAdminExecutionBlock;
 
-    mapping(address => bool) public pausers;
+    mapping(address => bool) public DEPRECATED_pausers;
 
     mapping(bytes32 => TaskStatus) public validatorManagementTaskStatus;
     uint16 validatorTaskBatchSize;
 
+    RoleRegistry public roleRegistry;
+
+    bytes32 public constant ETHERFI_ADMIN_ADMIN_ROLE = keccak256("ETHERFI_ADMIN_ADMIN_ROLE");
```

Comment:
I tried to keep the name convention consistent, but the resulting name for this contract isn't the best.
Lmk if I should break consistently for a better name